releasenotes/notes/vf2-layout-prune-4be07f26bdab23db.yaml

```diff
@@ -0,0 +1,16 @@
+---
+features_transpiler:
+  - |
+    :class:`.VF2Layout` now tracks partial candidate-layout scores during subgraph isomorphism
+    matching.  This has no meaningful effect when there is no perfect layout to find, but can
+    drastically reduce the cost of scoring layouts to choose the best candidate in high-symmetry
+    cases.
+upgrade:
+  - |
+    While the interface to :class:`.VF2Layout` remains logically the same, the ``max_trials``
+    argument now has much less effect as a runtime limiter, other than when set to the value ``1``.
+    This is because, with the new on-the-fly score-and-prune algorithm used internally by the class,
+    "complete" layouts are encountered far more rarely, and the count of "trials" only increments
+    when a new layout is encountered that is better in error rate than a previous one.  You should
+    instead use ``call_limit`` as the deterministic runtime limiter; where ``max_trials`` measures
+    complete layouts encountered, ``call_limit`` measures partial-layout extensions.
```

Comment:
It would be nice to explain what a reasonable value for ``call_limit`` looks like but this can be done in a follow-up adjustment to this release note.